doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+[id='migration-tool-niogit-7.4-proc_{context}']
+
+= Using the migration tool on a `.niogit` directory
+
+The `.niogit` directory of your {CENTRAL} distribution contains all {CENTRAL} data. You can use the {CENTRAL} migration tool on your `.niogit` directory to prepare project data and system configurations for migration to {PRODUCT} 7.5.
+
+
+.Prerequisites
+* {PRODUCT} 7.5 is installed.
+* {PRODUCT} 7.4 in installed and {CENTRAL} is not running on {PRODUCT} 7.4.
+* Java 8 is installed and is available on the path where the project migration tool will be installed.
+
+.Procedure
+
+. Locate the `bin/.niogit` directory in your {PRODUCT} 7.4 installation and locate the same directory in your new {PRODUCT} 7.5 installation.
+
++
+--
+Example:
+
+
+[source,subs="attributes+"]
+----
+$ ~/${PRODUCT_INIT_CAP}_7.4_HOME/bin/.niogit
+----
+
+[source,subs="attributes+"]
+----
+$ ~/${PRODUCT_INIT_CAP}_7.5_HOME/bin/.niogit
+----
+--
+. In your command terminal, copy the `.niogit` directory from your current {CENTRAL} distribution for {PRODUCT} 7.4 to the `bin` folder of the new {PRODUCT} 7.5 installation directory:
```

Comment:
That means overwriting the 7.5 directory that one located in the previous step?

Reply:
There should not be any .niogit directory if the 7.5 was not started before.